setup.py

```diff
@@ -17,9 +17,14 @@ def readme():
         return f.read()
 
 
-INSTALL_REQUIRES = (
-    ['pandas', 'httplib2', 'google-api-python-client', 'oauth2client']
-)
+INSTALL_REQUIRES = [
+    'pandas',
+    'httplib2>=0.9.2',
+    'google-api-python-client>=1.6.0',
+    'google-auth>=1.0.0',
+    'google-auth-httplib2>=0.0.1',
```

Comment:
I could try to tackle this using the the [conda-forge FAQ](https://github.com/conda-forge/staged-recipes#faq) and examples. I'm not familiar with conda-forge so this would allow me to get up to speed faster. In the first version I'm going to add you both as maintainers in the conda-forge recipe for google-auth @tswast @jreback , but please let me know if you'd like to be removed.